### DISCUSSION TITLE
Support for Azure Devops repositories

### DISCUSSION THREAD

**Discussion by pereirawill** [lindell/multi-gitter#354] [Q&A]
Would be possible to add support for azure DevOps repos please?
Thanks!

**Comment by lindell** [comment]
It is indeed possible :) I would be happy to merge a PR for it.

**Reply by Herve-M** [reply]
How should it be done? One PR and multiple gradual one?

**Reply by lindell** [reply]
Missed this comment when it was initially written. 

One PR with all basic functionality would be the way to go. It's basically to implement the VersionController interface
https://github.com/lindell/multi-gitter/tree/master/internal/scm
